identity/app/controllers/editprofile/ConsentsJourney.scala

```diff
@@ -48,7 +48,7 @@ trait ConsentsJourney
         returnUrlForm.bindFromRequest.fold(
           formWithErrors => Future.successful(BadRequest(Json.toJson(formWithErrors.errors.toList))),
           returnUrl => {
-            val newConsents = if (request.user.consents.isEmpty) Consent.defaultConsents else request.user.consents
+            val newConsents = if (request.user.consents.isEmpty) Consent.defaultConsents else Consent.addNewDefaults(request.user.consents)
```

Comment:
Adding this to today's catch up agenda to discuss.  Doesn't block this PR because the behaviour is not changed, but I want to discuss the intended behaviour.